### DISCUSSION TITLE
add keys-less updateMany to the js sdk

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#6822] [Feature Requests]
It's not well documented (I discovered it thanks to [this](https://github.com/directus/directus/discussions/5706#discussion-3367954)) but it seems that the _update many_ functionality can work without specifying the `keys` (user must provide a `query`, tho).

It'd be nice to have this functionality in the [js sdk](https://github.com/directus/directus/blob/b20a394f5130f9fb4d07bf3a77fcf7c912cfdd41/packages/sdk/src/base/items.ts#L55).

**Comment by rijkvanzanten** [comment]
I think we should add it as a dedicated `updateByQuery` method 🙂
